velox/common/memory/Allocation.cpp

```diff
@@ -30,12 +30,17 @@ Allocation::~Allocation() {
   }
 }
 
-void Allocation::append(uint8_t* address, int32_t numPages) {
```

Comment:
Can you add a test for this? Thanks!

Reply:
The `badNonContiguousAllocation` test has been modified to test this path.

Reply:
It might be better to have a dedicated test for AllocationTest.cpp? Thanks!